packages/url_launcher/url_launcher_macos/example/macos/RunnerTests/RunnerTests.swift

```diff
@@ -6,19 +6,121 @@ import FlutterMacOS
 import XCTest
 import url_launcher_macos
 
+/// A stub to simulate the system Url handler.
+class StubWorkspace: SystemURLHandler {
+
+  var isSuccessful = true
+
+  func open(_ url: URL) -> Bool {
+    return isSuccessful
+  }
+
+  func urlForApplication(toOpen: URL) -> URL? {
+    return toOpen
+  }
+}
+
 class RunnerTests: XCTestCase {
-  func testCanLaunch() throws {
+
+  func testCanLaunchSuccessReturnsTrue() throws {
     let plugin = UrlLauncherPlugin()
+
     let call = FlutterMethodCall(
       methodName: "canLaunch",
       arguments: ["url": "https://flutter.dev"])
-    var canLaunch: Bool?
+
+    plugin.handle(
+      call,
+      result: { (result: Any?) -> Void in
+        XCTAssertEqual(result as? Bool, true)
+      })
+  }
+
+  func testCanLaunchNoAppIsAbleToOpenUrlReturnsFalse() throws {
+    let plugin = UrlLauncherPlugin()
+
+    let call = FlutterMethodCall(
+      methodName: "canLaunch",
+      arguments: ["url": "example://flutter.dev"])
```

Comment:
It doesn't seem impossible that someone could have such a scheme when running tests locally. If you're not going to use the fake here, use a much more distinct scheme like `schemethatdoesnotexist`.

Reply:
The URL scheme has been changed.